requirements.txt

```diff
@@ -47,3 +47,4 @@ google-auth-oauthlib
 ############
 
 pytest
+typing-extensions
```

Comment:
Pretty sure this will break the lambda function as-is, since this isn't vendored in yet. Not sure how to do that - any suggestions?

If this is true, it also indicates a missing test. Shouldn't be too hard to write.

Reply:
I don't think it would matter, as the Lambda function never relies on any code that make use of these extensions. All that gets uploaded to Lambda are `lambda/handler.py`, `scanners/[scanner].py`, `utils/*.py`, and whatever vendored requirements are needed to support those things. 

Also,I think it would only be the flake8 or testing code path that would validate these things, and nothing that runs in Lambda.

Reply:
When running the code, all of the typing code is effectively a no-op. But we'd still have to be careful, since the `typing_extension` is still being imported at the head of `runner.py`.

If you say so, I'll believe you :)

Reply:
Yeah, `runner/runner.py` won't be imported by a Lambda function as packaged. During Lambda execution, a lot of code still runs locally -- all that gets run in Lambda is a scanner's `scan()` function, and the bare minimum supporting code to make that work. The CSV conversion, meta output, etc., all happens locally even when run in --lambda` mode.